### ISSUE TITLE
Availability of DigitalModel for one of the samples file

### ISSUE THREAD

**Issue by vincentmarchetti** [QualityInformationFramework/qif-community#26]
I am inquiring about the availability of the DigitalModel -- CAD file -- associated with one of the QIF sample files being distributed with this repository. The sample file in question is  samples/Results/QIF_PTS_SAMPLE.QIF, and the DigitalModel is referred to as CM14_Parts\QIF\QIF_pts_sample_SW2016.SLDPRT in a local file system.

I am developing, with the use of the QIF Python binding, 3D visualizations of QIF measurements in the X3D standard (ISO 14775), and am interested in adding a visualization of the original part geometry using established STEP -> X3D techniques to complement the visualization of measurement results.

Thank you for any assistance or information
Vince Marchetti
KShell Analysis & Web3D Consortium
vmarchetti@kshell.com

**Comment by bobstoneorigin** [comment]
Hi Vince,
I was able to find the QIF_pts_sample_SW2016 SolidWorks part file used to generate the QIF_PTS_SAMPLE QIF document but it contains no native SolidWorks entities. Our CheckMate addin would be needed to see anything.

The original source for the CMM program and results contained in that part file was a Zeiss Calypso inspection program written on Zeiss' cad cube training part. I've attached a STEP model of the Zeiss cad cube part oriented to match the QIF common coordinate system. I hope this will serve your purposes.
[cadcube_SW.zip](https://github.com/QualityInformationFramework/qif-community/files/9862644/cadcube_SW.zip)
